### PR TITLE
Update documentation with audit comments

### PR DESCRIPTION
**Pull request by mercepluka**
This PR reflects documentation changes for:
- Using ft_transfer_call and not ft_transfer
- LIX-01 (spending connectors resources if permissions are not set correctly)
- CON-05 (bridging assets while not possessing same Account ID on the destination side)
- MAC-04 (no automatic unused funds return)

### CODE REVIEW

File: docs/bridge/bridging/1_prerequisites.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+1. A Calimero shard needs to be running (in case you want to spin up your own Calimero shard send an [email](mailto:info@calimero.network) or [fill out this form](https://www.calimero.network/contact))
+2. All contracts need to be deployed on both NEAR and Calimero
+3. Relayer from Near to Calimero and a relayer from Calimero to Near need to be running
+4. Bridge service that monitors events in realtime on both Near and Calimero need to be running
+
+### Prerequisites for bridging FTs/NFTs
+
+When the Account is bridging a token (FT or NFT) from the source to the destination chain (for example from Calimero to Near) it should always obtain
```
Comment (ijerkovic) [inline docs/bridge/bridging/1_prerequisites.mdx:14]:
```suggestion
When an Account is bridging a token (FT or NFT) from the source to the destination chain (for example from Calimero to Near) it should always obtain
```

File: docs/bridge/bridging/1_prerequisites.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+the same Account ID on the destination chain. Let us show how Bob can lose his tokens if he does not obtain a destination account before bridging.
```
Comment (ijerkovic) [inline docs/bridge/bridging/1_prerequisites.mdx:15]:
```suggestion
the same Account ID on the destination chain beforehand. Let us show how Bob can lose his tokens if he does not obtain a destination account before bridging.
```

File: docs/bridge/bridging/1_prerequisites.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Bob has Account ID bob.testnet on the Calimero side and wants to bridge his FTs to the Near side. Bob successfully locks his FTs on the
+FT Connector on the Calimero side. Now Alice observes that Bob is bridging his assets to the Near side and creates bob.testnet Account on
+Near side (Bob did not possess this Account ID on the Near side before bridging). Now Alice gains Bob's bridged FTs and has successfully stolen them. Note that here Alice did not have any information that is not publicly available on Near side, so this problem can not be fixed by properly setting permissions.
+
+If you want to securely bridge assets from the source chain to the destination chain you should first possess the same Account ID on both sides, and
```
Comment (ijerkovic) [inline docs/bridge/bridging/1_prerequisites.mdx:21]:
```suggestion
If you want to securely bridge assets from the source chain to the destination chain you should *be in possession of the same Account ID on both sides*, and
```
See if this will bold the text to emphasize importance

File: docs/bridge/bridging/1_prerequisites.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+if that is not possible for some Account ID you should find another Account ID and obtain that new Account ID on both sides. Now you should
+bridge assets from a newly obtained Account ID.
+
+:::caution
+When bridging tokens you **MUST** possess the Account ID on the destination side. Assets **CAN** easily be stolen if you try to obtain the destination side Account ID after the bridging.
```
Comment (ijerkovic) [inline docs/bridge/bridging/1_prerequisites.mdx:26]:
```suggestion
When bridging tokens you **MUST** possess the Account ID on the destination side. Assets **CAN** be easily lost if you try to obtain the destination side Account ID after the bridging.
```

File: docs/bridge/bridging/2_permissions.mdx [hunk 5]
```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
...
+
+There are multiple reasons why bridge permissions need to be set properly:
+1. Avoid malicious entities from bridging FTs/NFTs into private shard
+2. Avoid leaking private information through Cross Shard Calls
+3. Avoid paying connectors gas fees on the destination chain for malicious actor's calls
+4. And potentially more...
+
+Now we will dig deeper into malicious actors' motivation for depleting connectors' funds on the public side (Near).
+When any bridging operation is called, the connector on the destination side initially pays gas fees for those calls. Funds spent
+on the gas are billed to the customer. Now we can see that if permissions are not properly set a malicious actor can initiate a large
```
Comment (ijerkovic) [inline docs/bridge/bridging/2_permissions.mdx:91]:
```suggestion
on the gas are paid by the shard owner. Now we can see that if permissions are not properly set a malicious actor can initiate a large
```

File: docs/bridge/bridging/2_permissions.mdx [hunk 6]
```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
...
+number of bridging operations and cause financial harm to the person or organization paying for the shard.
+
+Aside from causing financial harm to the shard payer, there is also a way to come into possession of a small amount of assets paid
```
Comment (ijerkovic) [inline docs/bridge/bridging/2_permissions.mdx:94]:
```suggestion
Aside from causing financial harm to the shard owner, there is also a way to come into possession of a small amount of assets paid
```

File: docs/bridge/bridging/2_permissions.mdx [hunk 7]
```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
...
+for malicious Cross Shard Calls. Let us look at the following scenario:
+1. There is some Calimero shard named `victim` and XSC permissions are set to allow all (allow rule `.*`) on the Calimero side
+2. An attacker with Account ID `attacker.testnet` gets permissions for interaction with Calimero shard. The attacker can get access to Calimero in a nuber of ways, including but not limited to shard being public, being regular member of the shard community, social engineering, and information leaking.
```
Comment (ijerkovic) [inline docs/bridge/bridging/2_permissions.mdx:97]:
```suggestion
2. An attacker with Account ID `attacker.testnet` gets permissions for interaction with Calimero shard. The attacker can get access to Calimero in a number of ways, including but not limited to: shard being public, being regular member of the shard community, social engineering, and information leaking.
```

File: docs/bridge/bridging/2_permissions.mdx [hunk 8]
```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
...
 - Management through the Calimero Bridge SDK
 - Directly through the CLI
+
+### Security and Motivation
+
+There are multiple reasons why bridge permissions need to be set properly:
+1. Avoid malicious entities from bridging FTs/NFTs into private shard
+2. Avoid leaking private information through Cross Shard Calls
+3. Avoid paying connectors gas fees on the destination chain for malicious actor's calls
+4. And potentially more...
```
Comment (ijerkovic) [inline docs/bridge/bridging/2_permissions.mdx:87]:
```suggestion
```
I would either rephrase it, or remove it, as it sounds we are hiding some other use cases

File: docs/bridge/bridging/2_permissions.mdx [hunk 9]
```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
...
+
+Aside from causing financial harm to the shard payer, there is also a way to come into possession of a small amount of assets paid
+for malicious Cross Shard Calls. Let us look at the following scenario:
+1. There is some Calimero shard named `victim` and XSC permissions are set to allow all (allow rule `.*`) on the Calimero side
+2. An attacker with Account ID `attacker.testnet` gets permissions for interaction with Calimero shard. The attacker can get access to Calimero in a nuber of ways, including but not limited to shard being public, being regular member of the shard community, social engineering, and information leaking.
+3. The attacker deploys a smart contract on the Near side called `contract.testnet`, this contract is just a random contract with some method that uses gas
+4. The attacker initiates a large number of Cross Shard Calls from the Calimero side to the Near side for a contract `contract.testnet`
+5. An attacker gets 30% of used gas (gas paid by Cross Shard Connector on the Near side) as a reward for deploying a smart contract. [Link to Near gas documentation](https://docs.near.org/concepts/basics/transactions/gas)
```
Comment (ijerkovic) [inline docs/bridge/bridging/2_permissions.mdx:100]: I don't think that in current XSC setup, new contracts can be deployed on the other side directly, you could call a method that deploys a new contract, in that way this could happen, can you rephrase it?

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 10]
```diff
@@ -0,0 +1,14 @@
+---
+title: Return of unused assets
+sidebar_position: 5
+---
+
+If user locks assets (FTs or NFTs) on the source side connector and there is some kind of problem in destination side or bridge itself
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:6]:
```suggestion
If user locks assets (FTs or NFTs) on the source chain connector and there is some kind of problem on the destination chain, or the automatic bridge service is experiencing downtime (which should happen extremely rarely, but in reality can happen)
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+bridging will not work. If asset is not bridged in the following 4 days from the initial locking transaction asset will no longer be
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:7]:
```suggestion
,bridging will not work. If asset is not bridged in the following 96 hours from the initial locking transaction, asset will no longer be
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+eligible for bridging to the destination side.
+
+Currently user with this problem should contact a shard admin, admin should verify that asset can no longer be bridged and return assets to
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:10]:
```suggestion
Currently user with this problem should contact a shard admin. Admin should verify that asset can no longer be bridged and return assets to
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+the original owner (user). Note that if admin unlocks it before 4 days pass user will effectively get his assets two times since he can
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:11]:
```suggestion
the original owner (user). Note that if the admin unlocks it before 4 days pass user will effectively get his assets two times since he can
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+still prove that he locked assets (and bridge it to destination side), and he will get locked assets on the source side back.
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:12]:
```suggestion
still prove that he locked assets (and bridge it to destination chain), and he will get locked assets on the source chain back as well.
```

**Review by ijerkovic — CHANGES_REQUESTED** [review]
added some tweaks to the wording

File: docs/bridge/bridging/1_prerequisites.mdx (hunk 3 above)
Comment (mercepluka) [reply docs/bridge/bridging/1_prerequisites.mdx]: Fixed, `**` is for bolding the text, that is the reason I did not just commit this suggestion

File: docs/bridge/bridging/2_permissions.mdx (hunk 9 above)
Comment (mercepluka) [reply docs/bridge/bridging/2_permissions.mdx]: I rephrased this as agreed on the call, please check if it is properly phrased

File: docs/bridge/bridging/1_prerequisites.mdx [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+the same Account ID on the destination chain beforehand. Let us show how Bob can lose his tokens if he does not obtain a destination account before bridging.
+
+Bob has Account ID bob.testnet on the Calimero side and wants to bridge his FTs to the Near side. Bob successfully locks his FTs on the
+FT Connector on the Calimero side. Now Alice observes that Bob is bridging his assets to the Near side and creates bob.testnet Account on
+Near side (Bob did not possess this Account ID on the Near side before bridging). Now Alice gains Bob's bridged FTs and has successfully stolen them. Note that here Alice did not have any information that is not publicly available on Near side, so this problem can not be fixed by properly setting permissions.
+
+If you want to securely bridge assets from the source chain to the destination chain you should **be in possession of the same Account ID on both sides**, and
+if that is not possible for some Account ID you should find another Account ID and obtain that new Account ID on both sides. Now you should
+bridge assets from a newly obtained Account ID.
```
Comment (ijerkovic) [inline docs/bridge/bridging/1_prerequisites.mdx:23]:
```suggestion
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 16]
```diff
@@ -0,0 +1,14 @@
+---
+title: Return of unused assets
+sidebar_position: 5
+---
+
+If user locks assets (FTs or NFTs) on the source chain connector and there is some kind of problem on the destination chain, or the automatic bridge service is experiencing downtime (which should happen extremely rarely, but in reality can happen)
+,bridging will not work. If asset is not bridged in the following 96 hours from the initial locking transaction, asset will no longer be
+eligible for bridging to the destination side.
+
+Currently user with this problem should contact a shard admin. Admin should verify that asset can no longer be bridged and return assets to
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:10]:
```suggestion
Currently user with this problem should contact the shard admin. The admin should verify that asset can no longer be bridged and return assets to
```

File: docs/bridge/bridging/5_return_unused_assets.mdx [hunk 17]
```diff
@@ -0,0 +1,14 @@
+---
+title: Return of unused assets
+sidebar_position: 5
+---
+
+If user locks assets (FTs or NFTs) on the source chain connector and there is some kind of problem on the destination chain, or the automatic bridge service is experiencing downtime (which should happen extremely rarely, but in reality can happen)
+,bridging will not work. If asset is not bridged in the following 96 hours from the initial locking transaction, asset will no longer be
```
Comment (ijerkovic) [inline docs/bridge/bridging/5_return_unused_assets.mdx:7]: nit: space after ,

**Review by ijerkovic — APPROVED** [review]
